src/containers/ProfilePage/ProfilePage.js

```diff
@@ -142,6 +151,50 @@ export const DesktopReviews = props => {
   );
 };
 
+export const CustomUserFields = props => {
+  const { publicData, metadata, userFields } = props;
+  return (
+    <>
+      <H4 as="h2" className={css.listingsTitle}>
+        <FormattedMessage id="ProfilePage.customFieldsHeading" />
+      </H4>
```

Comment:
Wireframes don't have this heading included. So, perhaps it's better to omit it.

It's also a bit odd case from the heading structure:

H1: "Hello, I'm Juho M"

  H2: Details
  H2: SectionMultiEnumMaybe / SectionTextMaybe (all of the fields)

If we add an extra heading between H1 and sections, we should add H2 for the customFieldsHeading and H3 for the sections:
H1: "Hello, I'm Juho M"
  H2: customFieldsHeading
    H3: Details
    H3: SectionMultiEnumMaybe / SectionTextMaybe (all of the fields)

--

However, I don't think the extra heading for the custom fields would serve any meaningful purpose since the bio & custom fields could just tell the same story.


H1: "Awesome Pasta Joint"
p: "Our restaurant is located in central Helsinki and serves fresh pasta. We make pasta dough on the morning. [etc...]"

H2 (text): Origin story
p: "We came up with the idea of creating own restaurant during the pandemic [etc...]"

H2 (multi-enum): Pasta types we use
- Spaghetti
- Penne
- Riggatone
- Etc...

Reply:
So, I'd suggest removing this heading (and then check that the margins work without it).

Reply:
Heading removed!